Vizzuality/heco-invest
# feat: Favourite Investor API endpoint

New API endpoint which allows to mark/unmark investors as favourite. Controller is basically same as ones used for marking Project or Project Developers as favourite.

I have moved all policy checks to cancancan for these controllers, therefore You are now required to have approved account otherwise you receive forbidden error.

## Testing instructions

New API endpoint for making Investors favourite is available via Rswag documentation, therefore everything can be tested via there. Please make sure to use approved account, otherwise you should receive forbidden error.

## Tracking

https://vizzuality.atlassian.net/browse/LET-526
https://vizzuality.atlassian.net/browse/LET-527
https://vizzuality.atlassian.net/browse/LET-554
